numba/core/options.py

```diff
@@ -88,6 +88,7 @@ class DefaultOptions:
     fastmath = _mapping("fastmath")
     error_model = _mapping("error_model")
     inline = _mapping("inline")
+    forceinline = _mapping("forceinline")
```

Comment:
NOTE: the `forceinline` flag pre-existed this PR. It was previously internal only and it is used by the `FunctionDescriptor.inline` at https://github.com/numba/numba/blob/58426cf7658177b792b820ff66c0945985b72da0/numba/core/typed_passes.py#L390